src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -315,18 +315,22 @@
                                     :binding "mod+shift+p"
                                     :fn      (fn [] (state/toggle! :ui/command-palette-open?))}
 
-   :graph/open              {:desc    "Select graph to open"
+   :graph/open                     {:desc    "Select graph to open"
                                     :fn      (fn [] (state/set-state! :ui/open-select :graph-open))
                                     :binding "mod+shift+g"}
 
-   :graph/remove            {:desc    "Remove a graph"
+   :graph/remove                   {:desc    "Remove a graph"
                                     :fn      (fn [] (state/set-state! :ui/open-select :graph-remove))
                                     :binding false}
 
    :graph/add                      {:desc "Add a graph"
                                     :fn (fn [] (route-handler/redirect! {:to :repo-add}))
                                     :binding false}
 
+   :graph/save                     {:desc "Save current graph to disk"
+                                    :fn #(state/pub-event! [:graph/save])
```

Comment:
It would be nice if users could define commands in config.edn to route to any event. I've updated https://quire.io/w/Logseq-_team_dedicated_to_sport_of_knowledge/189/Define_shortcuts_commands_in_config to capture this